src/main/java/racingcar/domain/Car.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.domain;
+
+import static racingcar.util.CarNameValidationUtil.*;
+
+public class Car {
+
+    private final String name;
+    private int position;
+
+    private final int MIN_LENGTH_OF_NAME = 1;
+    private final int MAX_LENGTH_OF_NAME = 5;
+    private final int CONDITION_FOR_MOVE = 4;
+
+    public Car(String name) {
+        validateCarName(name, MIN_LENGTH_OF_NAME, MAX_LENGTH_OF_NAME);
+        this.name = name;
+        this.position = 0;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public int getPosition() {
+        return position;
+    }
+
+    public void moveOrNot(int target) {
```

Comment:
자동차에게 이동하라 메시지만 보내고 이동할지 말지는 자동차 스스로 판단하여 이동한다고 생각했을 때
moveOrNot이라는 메서드명은 구현에 대한 내용이 조금 내포되어 있는 것 같아요. 아리는 어떻게 생각하시나요?
```suggestion
    public void move(int target) {
```

Reply:
우선 제가 "moveOrNot" 이라는 메서드명을 사용한 이유부터 설명하겠습니다.
메서드명은 해당 메서드가 처리하는 일을 함축적으로 전달하는 것이라고 이해하고 있습니다.
따라서 메서드명에서 "자동차가 이동할지 말지 스스로 판단하여 이동시키는 함수" 라는 의미가 드러나기를 바랬습니다. 
하지만 단순히 move만을 사용한다면 그저 "자동차를 이동시키는 함수" 라는 의미만이 전달될 것 같습니다.

"구현에 대한 내용이 내포되어있는 것 같아요" 라는 문장을 조금만 더 자세히 설명해주실 수 있을까요?
메서드명에 구현에 관련된 내용이 내포되는 것이 적절하지 않다는 말씀이신지, 어떻게 이동시키든 저 메서드가 알아서 하는 것이니 "자동차를 이동시키는 함수"정도의 의미만 표현해도 충분하다는 말씀이신지, 혹은 제가 이해하지 못한  다른 뜻이 있는 것인지 궁금합니다!

Reply:
후자의 의미로 코멘트를 남긴 거예요 :)


만약 요구사항이 변경되어 입력 range 마다 이동해야할 거리가 정해진 경우 각 케이스에 대한 모든 것을 메서드명으로 나타내야 할까요?

Reply:
> 메서드명에 구현에 관련된 내용이 내포되는 것이 적절하지 않다는 말씀이신지, 어떻게 이동시키든 저 메서드가 알아서 하는 것이니 "자동차를 이동시키는 함수"정도의 의미만 표현해도 충분하다는 말씀이신지, 혹은 제가 이해하지 못한 다른 뜻이 있는 것인지 궁금합니다!

둘 다 해당 됩니다.

만약 요구사항이 변경되어 입력 range 마다 이동해야할 거리가 정해진 경우 각 케이스에 대한 모든 것을 메서드명으로 나타내야 할까요?

Reply:
요구사항이 변경에 따라 메서드명이 변경 될 수 있다는 생각은 못했습니다.. move로 변경했습니다!